docs/guides-use-frame.md

```diff
@@ -4,10 +4,106 @@ title: How to use Frame
+</p>
+
+To take full advantage of Frame we recommend you have the `aragonCLI` installed as well. If you already have it, skip to the next section, otherwise ran:
+
+```sh
+npm install -g @aragon/cli
+```
+
+> You need a Ledger or Trezor wallet to use Frame.
+
+## Configure you Device
+
+#### Ledger
+
+On Ledger devices insert your pin on the device to unlock it and then open Frame.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-ledger.gif">
+</p>
+
+#### Trezor
+
+On Trezor devices insert your pin directly on Frame.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-trezor.gif">
+</p>
+
+You are ready to select the network and account you would will use.
+
+#### Network
+
+To choose the appropiate network select the button on the uper right corner (the one with the three arrows). This page is the app menu, here you can configure general settings for Frame. We will focus on the Connection option. Right now the default is Rinkeby but you can aslo choose Mainnet, Ropsten and Kovan. For this tutorial we will select Rinkeby.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-app-menu.gif">
+</p>
+
+#### Accounts
+
+The last step is to choose your account, click on the configuration button, the one on the right, and then click on the accounts button that will pop up. This action will show you a new menu with the avaible accounts on your device for the network. Select one, be sure to have some test ether, get some on the [Rinkeby faucet](https://faucet.rinkeby.io) otherwise.
+
+<p align="center">
+  <img width="240" height="500" src="/docs/assets/frame/frame-accounts.gif">
+</p>
+
+In the configuration menu you can also manage the dapp permisions you have granted for that particular account. We will explain more about this on the next section.
+
+Now that we finish setting up the device we are ready to sign our first transaction with Frame.
+
+## Signing your first transaction
+
+On this section we are going to interact with an already [deployed DAO on rinkeby](https://rinkeby.aragon.org/#/0x16b3C84d4DB149590981F2d3A36e14Db96069730/) that allow anyone to mint any amount of `FRAME` tokens to any address. We will use an `aragonCLI` command to execute a function on the `token-manager` app of the DAO. You might find helpful to [learn more about `dao exec` command](https://hack.aragon.org/docs/cli-dao-commands#dao-exec) before continue.
+
+The first step is to ran the command on the terminal:
+
+```sh
+dao exec 0x16b3C84d4DB149590981F2d3A36e14Db96069730 0x1A8D8BB7eB5aC6E6F51dF5E65d301c7e5dD00D58 mint <address> <amount> --environment aragon:rinkeby --use-frame
+```
+
+Under `<address>` input an address of your choice and under `<amount>` provide the amount of tokens you would like to mint for the address. The options `--environments` and `--use-frame` tell the `aragonCLI` that we want to interact with rinkeby and we will use Frame as signer.
+
```

Comment:
Would be useful to note down what the `0x16b3C84d4DB149590981F2d3A36e14Db96069730` and `0x1A8D8BB7eB5aC6E6F51dF5E65d301c7e5dD00D58` parts of the command correspond to in case the person hasn't read up on the `dao exec` command

Reply:
Definitly 👍